docker/sf-gpu.Dockerfile

```diff
@@ -1,3 +1,28 @@
 FROM tensorflow/tensorflow:2.10.1-gpu
-
+#install secretflow
+#you are supposed to add the mirror source of pypi to accelerate installation of SecretFlow and accelerate the building of images
+#if not, the building of images are prone to fail very much 
 RUN pip install -U secretflow
+# remove the cpu version of jax and jaxlib which were installed when run 'pip install -U secretflow'
+RUN pip uninstall -y jax
+RUN pip uninstall -y jaxlib
+#install the dependencies of cuda11
+#you are supposed to  add the mirror source of pypi to accelerate installation of nvidia packages of cuda11, 
+#if not, the building of images are prone to fail very much 
+RUN pip install nvidia-cublas-cu11
```

Comment:
Can you merge all pip install into one `RUN` to reduce number of layers in the final docker image?